### DISCUSSION TITLE
Vontar X2(w2) add dtb

### DISCUSSION THREAD

**Discussion by Adney** [devmfc/debian-on-amlogic#50] [Ideas]
Fixed the framebuffer address - HDMI is working

[meson-s4-vontarw2dtb.zip](https://github.com/devmfc/debian-on-amlogic/files/15153973/meson-s4-vontarw2dtb.zip)

WI-fi started working with the HK6334Q chip (I had to upload bin and txt). . But they also come across with another wi-fi chip.

**Comment by devmfc** [comment]
Thanks! 

I was just working on a new release, so I added a version of the Vontar X2 to the box configs based on your modifications. Do you know which VFD display controller your box has? TM1628 or FD655? It would be nice to add support for that too.

**Comment by Adney** [comment]
> Do you know which VFD display controller your box has?

**FD655**

 Drivers Wi-fi/bt HK6334Q
[brcmfmac4334.zip](https://github.com/devmfc/debian-on-amlogic/files/15158192/brcmfmac4334.zip)

**Comment by TimurMo** [comment]
try loading box=vontarx2
gives an error message. I have a TVbox 4/32
Maybe that's why it doesn't fit?

or tell me how to get the dtb file out of android
![1717757491976](https://github.com/devmfc/debian-on-amlogic/assets/10673495/0f1a0ad5-89a0-4fdc-b13d-858c7ffeec42)
![1717757491983](https://github.com/devmfc/debian-on-amlogic/assets/10673495/7ba051d5-b8d4-4a59-8d8a-6c0a21cd9961)

**Comment by TimurMo** [comment]
The problem turned out to be in the SD card. I recorded it on a USB flash drive and downloaded it from it.
